uds/translator/configurable_translator.py

```
@@ -0,0 +1,1002 @@
+    def diagnostic_session_type_mapping(self, value: Mapping[int, str]) -> None:
+        """
+        Set diagnosticSessionType (SubFunction of DiagnosticSessionControl) value to name mapping.
+
+        :param value: Mapping value to set.
+        """
+        diagnostic_session_control = self.services_mapping[RequestSID.DiagnosticSessionControl]
+        diagnostic_session_control.request_structure[0].children[1].values_mapping = value
+        diagnostic_session_control.response_structure[0].children[1].values_mapping = value
+
+    @property
+    def reset_type_mapping(self) -> None | Mapping[int, str]:
```

Ruff (0.16.1)

[warning] `None` not at the end of the type union.

Move `None` to the end of the type union

(RUF036)